pkg/model/dev.go

```diff
@@ -1299,12 +1299,15 @@ func getLocalhost() string {
 // Copy clones the buildInfo without the pointers
 func (b *BuildInfo) Copy() *BuildInfo {
 	result := &BuildInfo{
```

Comment:
I think we will always forget to add the new fields here... Maybe we should use something like https://github.com/jinzhu/copier to perform this task. The problem with these modules is that they use reflect inside which can cause runtime errors.

Reply:
i've used marshall unmarshall method, is this an option? seems to work fine

Reply:
@teresaromero we could also have a silent error with marshalling because it needs the fields to be exported. If some day we add an unexported field, it won't be copied

Reply:
i've reverted to loop between fields and pointers.
@jLopezbarb it is a risk to forget... but this struct is not changing that much, and eventually the bug will appear when we use differnt build options if we add some new.... I prefer this than other silent errors or adding a new library....

Reply:
I'm ok with it, I just wanted to make sure that this was going to be a problem when implementing new fields. Right now it only affects buildInfo so it's a thing we can deal with it but if we ever have several structs to copy we should take a look for one of the solutions